composer.json

```diff
@@ -53,5 +54,8 @@
         }
     },
 
+    "prefer-stable": true,
+    "minimum-stability": "dev",
```

Comment:
Well, first things first...

Or what can be done, as it is done in the travis configuration, it's to enable these only in travis (or in dev through a global configuration ?). So it would mean to remove these two configuration options. Don't forget that it's only for dev-mode...